parent/pom.xml

```diff
@@ -1155,6 +1155,53 @@
                     </bottom>
                 </configuration>
             </plugin>
+
+            <!--
+                Plugin to remove org.apache.ignite.* artifacts from local repository before the build
+                (always clean build with current reactor dependency resolution)
+            -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>purge-local-repository</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>purge-local-repository</goal>
+                        </goals>
+                        <configuration>
+                            <manualIncludes>
+                                <manualInclude>org.apache.ignite</manualInclude>
```

Comment:
Won't this also remove dependencies from Ignite 2? Should we use `include` instead?